aidial_adapter_dial/utils/sse_stream.py

```diff
@@ -68,13 +30,14 @@ async def to_openai_sse_stream(
     try:
         async for chunk in stream:
             yield format_chunk(chunk)
-        yield END_CHUNK
     except Exception as e:
         log.exception(
             f"caught exception while streaming: {type(e).__module__}.{type(e).__name__}"
         )
 
         dial_exception = to_dial_exception(e)
-        starlette_exception = to_starlette_exception(dial_exception)
+        error_chunk = to_json_content(dial_exception)
+
+        yield format_chunk(error_chunk)
 
-        yield format_chunk(starlette_exception.detail)
+    yield END_CHUNK
```

Comment:
Q: Before this PR you returned END_CHUNK only in case there is no error. Now it's returned even after exception handling. Is this intentional?
A: we always return CHUNK_CHUNK in DIAL SDK, so I reasoned that we should follow the same logic everywhere

Q: If it's intentional, why you didn't use else/finally there?
A: I don't think we need to use finally here. Imagine `SystemExit`, `CancelError` or `KeyboarInterrupt` exception is thrown _(none of which is `Exception`)_, the error handler won't catch it, but we will send END_CHUNK. This is strange. I would rather interrupt the response abruptly than make a false impression that everything is ok.